lib/fs.js

```diff
@@ -1353,7 +1373,10 @@ function FSWatcher() {
   this._handle.onchange = function(status, event, filename) {
     if (status < 0) {
       self._handle.close();
-      const error = errnoException(status, `watch ${filename}`);
+      const error = !filename ?
+          errnoException(status, 'Error watching file for changes:') :
+          errnoException(status,
+                         `Error watching file ${filename} for changes:`);
```

Comment:
maybe just break it out into its own commit? May be small, but in case one needs to be rolled back for some unknown reason it won't affect the other.